### PR TITLE
Add ipv6 support

### PR DESCRIPTION
**Pull request by unilynx**
Add ipv6 support
Remove the custom regular expression for IP validation and use node's standard net.isIP

### CODE REVIEW

File: test/basic.js [hunk 1]
```diff
@@ -53,6 +53,24 @@ test('set', function (t) {
...
+test('set ipv6', function (t) {
+  t.plan(4)
+  hostile.set('::1', 'peercdn.com', function (err) {
+    t.error(err)
+    hostile.get(false, function (err, lines) {
+      t.error(err)
+      var exists = lines.some(function (line) {
+        return line[0] === '::1' && line[1] === 'peercdn.com'
+      })
+      t.ok(exists, 'ipv6 line was not added')
```
Comment (feross) [inline test/basic.js]: The message here should describe what you're checking for, not what happened in the failure case.

File: test/basic.js [hunk 2]
```diff
@@ -53,6 +53,24 @@ test('set', function (t) {
...
+      exists = lines.some(function (line) {
+        return line[0] === '127.0.0.1' && line[1] === 'peercdn.com'
+      })
+      t.ok(exists, 'ipv4 line was incorrectly removed')
```
Comment (feross) [inline test/basic.js]: Same

**Comment by feross** [comment]
@unilynx Would you like to be a maintainer of this package?

**Comment by feross** [comment]
Released as 1.3.0

**Comment by unilynx** [comment]
I see you've already made the changes you requested in your comments. About being a maintainer, sure, if you need a hand with the package. I use it almost daily anyway.. (but generally only on osx, so other platforms are harder to test(

**Comment by feross** [comment]
@unilynx Added you as a collaborator!

**Pull request merged**